localization/ch.xml

```diff
@@ -1,57 +1,57 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <localizationPack name="Switzerland" version="1.0">
 	<currencies>
-		<currency name="Franc" iso_code="CHF" iso_code_num="756" sign="CHF" blank="1" conversion_rate="1.34183" format="5" decimals="1" />
+		<currency name="Swiss Franc" iso_code="CHF" iso_code_num="756" sign="CHF" blank="1" conversion_rate="1.34183" format="5" decimals="1" />
 	</currencies>
 	<languages>
 		<language iso_code="de" />
 		<language iso_code="fr" />
 		<language iso_code="it" />
 	</languages>
 	<taxes>
-		<tax id="1" name="TVA CH 7.7%" rate="7.7" />
-		<tax id="2" name="TVA CH 3.7%" rate="3.7" />
-		<tax id="3" name="TVA CH 2.5%" rate="2.5" />
+		<tax id="1" name="VAT CH 7.7%" rate="7.7" />
+		<tax id="2" name="VAT CH 2.5%" rate="2.5" />
+		<tax id="3" name="VAT CH 3.7%" rate="3.7" />
 
-		<taxRulesGroup name="CH Standard Rate (7.7%)">
+		<taxRulesGroup name="CH Normal Rate (7.7%)">
 			<taxRule iso_code_country="ch" id_tax="1" />
 		</taxRulesGroup>
 
-		<taxRulesGroup name="CH Reduced Rate (3.7%)">
+		<taxRulesGroup name="CH Reduced Rate (2.5%)">
 			<taxRule iso_code_country="ch" id_tax="2" />
 		</taxRulesGroup>
 
-		<taxRulesGroup name="CH Super Reduced Rate (2.5%)">
+		<taxRulesGroup name="CH Special Rate (3.7%)">
```

Comment:
We have a way to refer to VAT tax rates, it is the same for all countries. So I'd prefer to keep it harmonized. Is there any reason you want to call those taxes that way? cf. https://www.avalara.com/vatlive/en/country-guides/europe/switzerland/swiss-vat-rates.html